packages/client/src/pages/leaderboard/userBox/model/effects.ts

```diff
@@ -0,0 +1,24 @@
+import { createEffect } from 'effector';
+import { practicumApi } from 'shared/api/api';
+import { AxiosError } from 'axios';
+import { GetUserByIdGateProps } from 'pages/leaderboard/types';
+
+export const getUserByIdFx = createEffect<
+  GetUserByIdGateProps,
+  void,
+  AxiosError
+>(async ({ userId, userFn, loadingFn }) => {
+  const { data } = await practicumApi.user.userDetail(userId);
+
+  // Имитация задержки при получении данных
+  const delay = Math.random() * 4000 + 500;
```

Comment:
Я так понимаю имитация нужна для режима разработки, значит нужно сделать 

```suggestion
  if (isDev) {
     const delay = Math.random() * 4000 + 500;
```